feature/home/src/main/java/com/epmedu/animeal/home/presentation/HomeScreenUI.kt

```diff
@@ -147,11 +147,7 @@ internal fun HomeScreenUI(
             HomeMapbox(
                 state = state,
                 onFeedingPointSelect = { onScreenEvent(FeedingPointEvent.Select(it)) },
-                onMapInteraction = {
-                    if (bottomSheetState.isExpanding && state.feedingRouteState is FeedingRouteState.Disabled) {
-                        scope.launch { bottomSheetState.show() }
-                    }
-                },
+                onMapInteraction = { /* do nothing */ },
```

Comment:
I tried to understand why that code needs, and unfortunately, I didn't understand. But attempts to display the bottomSheet before the state are ready, it's a behavior that led to crash.

Anyway, from my perspective, it would be great to know what that logic does,  since I can't find any cases when it would be used. 

So, for that scenario, the UnitTests could be useful, but we haven't any tests here.

Reply:
It collapses bottom sheet when user drags the map

https://user-images.githubusercontent.com/83027107/234281995-862cb400-7d6a-4adb-99ae-2d9553570927.mp4